presto-main/src/main/java/io/prestosql/operator/scalar/timetz/TimeWithTimeZoneToVarcharCast.java

```diff
@@ -48,18 +48,17 @@ public static Slice cast(@LiteralParameter("p") long precision, @SqlType("time(p
         long nanos = unpackTimeNanos(packedTime);
         int offsetMinutes = unpackOffsetMinutes(packedTime);
 
-        return formatAsString((int) precision, nanos * PICOSECONDS_PER_NANOSECOND, offsetMinutes);
+        return toVarchar((int) precision, nanos * PICOSECONDS_PER_NANOSECOND, offsetMinutes);
     }
 
     @LiteralParameters({"x", "p"})
     @SqlType("varchar(x)")
     public static Slice cast(@LiteralParameter("p") long precision, @SqlType("time(p) with time zone") LongTimeWithTimeZone time)
     {
-        return formatAsString((int) precision, time.getPicoSeconds(), time.getOffsetMinutes());
+        return toVarchar((int) precision, time.getPicoSeconds(), time.getOffsetMinutes());
     }
 
-    // Can't name this format() because we can't have a qualified reference to String.format() below
-    private static Slice formatAsString(int precision, long picos, int offsetMinutes)
+    private static Slice toVarchar(int precision, long picos, int offsetMinutes)
```

Comment:
I intentionally didn't name it `toVarchar` when I wrote this because I wanted to convey the notion that this is about formatting the value in a string-like format. The fact that it's a varchar is a concern for the caller. See my comment below -- I think the fact that we have to work around naming (and this is not the only case) just to satisfy some convention around imports shows why this "rule" shouldn't be applied globally.